CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed
+
+- Error where setting `PlaybackConfig.isAutoplayEnabled = true` causes the player view creation to fail on Android
+
```

Comment:
Nit -- Wording: isn't the crash happening when the flag is not set?

```suggestion
- Error where not setting `PlaybackConfig.isAutoplayEnabled` causes the player view creation to fail on Android

```

Reply:
No, the crash happens on the `isBackgroundPlaybackEnabled` flag if `isAutoplayEnabled` is set to true.

Reply:
Ah I see, thanks for clarifying